src/FeedContext/Google/Shopping/ProductItemContext.php

```diff
@@ -60,8 +68,13 @@ public function getContextList(object $product, ChannelInterface $channel, Local
             ));
         }
 
-        $productType = $this->getProductType($product);
+        if ($product instanceof ProductTypeAwareInterface) {
```

Comment:
The name 'Product type' comes from Google's lingo. If we should have an interface like this, it should be named something else: 'Breadcrumb', 'Taxon Path' or something. I probably like taxon path the most

Reply:
Ok